SPECS/kubevirt/Allocate-2-cpu-for-the-emulator-thread.patch

```diff
@@ -15,20 +15,26 @@ diff --git a/pkg/virt-controller/services/renderresources.go b/pkg/virt-controll
 index 46b242932..d75354bc5 100644
 --- a/pkg/virt-controller/services/renderresources.go
 +++ b/pkg/virt-controller/services/renderresources.go
-@@ -194,9 +194,11 @@ func WithCPUPinning(cpu *v1.CPU) ResourceRendererOption {
+@@ -212,9 +212,11 @@ func WithCPUPinning(cpu *v1.CPU) ResourceRendererOption {
  			}
  		}
  
--		// allocate 1 more pcpu if IsolateEmulatorThread request
+-		// allocate pcpus for emulatorThread if IsolateEmulatorThread is requested
 +		// allocate 2 more pcpu if IsolateEmulatorThread request
 +		// the extra cpu ensures hyperthread SMT cpu alignment when cpu-manager
 +		// policy full-pcpus-only is used.
  		if cpu.IsolateEmulatorThread {
--			emulatorThreadCPU := resource.NewQuantity(1, resource.BinarySI)
-+			emulatorThreadCPU := resource.NewQuantity(2, resource.BinarySI)
- 			limits := renderer.calculatedLimits[k8sv1.ResourceCPU]
- 			limits.Add(*emulatorThreadCPU)
- 			renderer.vmLimits[k8sv1.ResourceCPU] = limits
+-			emulatorThreadCPUs := resource.NewQuantity(1, resource.BinarySI)
++			emulatorThreadCPUs := resource.NewQuantity(2, resource.BinarySI)
+
+			limits := renderer.calculatedLimits[k8sv1.ResourceCPU]
```

Comment:
Not sure about this, we only need to make the quality(2)

Reply:
Looks like we shouldn't need the patch at all given this check `emulatorThreadCompleteToEvenParityAnnotationExists` , the right fix would be to add new annotations

Reply:
to confirm, I dont need to do any other fix other than removing this patch?  FYI @kanikanema

Reply:
correct. We should note that though for CAPKV migration to the new version.